lib/r10k/cli/deploy.rb

```diff
@@ -10,6 +10,15 @@
 
 module R10K::CLI
   module Deploy
+
+    class TransformExcludeSpec
```

Comment:
I think a method is meant for more general usage, but since this is a specific instance to be used for the exclude_spec setting, I think a class is more appropriate.